src/domains/wallet-common/env/weu-prod/terraform.tfvars

```diff
@@ -75,3 +76,13 @@ redis_wallet_params = {
 }
 
 enable_iac_pipeline = true
+
+wallet_storage_params = {
+  enabled                       = true
+  tier                          = "Standard"
+  kind                          = "StorageV2"
+  account_replication_type      = "GZRS",
+  advanced_threat_protection    = true,
+  retention_days                = 30,
```

Comment:
Are 30 retention days a valid values for wallet use case or maybe can be lowered to 7?

Reply:
Yeah, maybe even 3 could be enough, for now I'll lower to 7

Reply:
Changed in 3abd91c4